cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
newSession is anyway creating a new Object. What purpose this explicit object creation is solving? We added newSession instead of explicit object creation to share the UI tab.  IMO, we should leave this code untouched.